app/uk/gov/hmrc/taxenrolmentassignmentfrontend/views/ReportSuspiciousID.scala.html

```diff
@@ -86,14 +88,15 @@ <h1 class="govuk-heading-xl">@{messages(s"ReportSuspiciousID.heading")}</h1>
 content = HtmlContent(content)
 ))
 
-@if(currentAccountHasSA) {
-<p class="govuk-body">@messages("ReportSuspiciousID.sa.text1") </p>
-<br>
-@govukButton(Button(
-content = Text(messages("common.continue")),
-href = Some("?")
-))
-}
-
-
+@if(saOnOtherAccount) {
+  @formHelper(action = routes.ReportSuspiciousIDController.continue) {
+    <p class="govuk-body">@messages("ReportSuspiciousID.sa.text1") </p>
```

Comment:
Isn't this wrong? Shouldn't it have stayed as '`currentAccountHasSA'`? The message key is for the message - 

> You can continue to your personal tax account with the user ID you are currently signed in with.

Reply:
No this is reporting fraud after going through the SA on another account journey

Reply:
There are two ways to get to the report fraud page. Either for an unknown account having the PT or SA enrolment. So If the user has come through the PT report journey, but has SA on another account, this content would appear and be invalid. If anything, this text shouldn't be dependent on SA but rather the PT enrolment.

Reply:
Yes an unknown account not the current account

Reply:
It is the saOnOtherAccount journey that this button is shown for.

Reply:
I don't know if I've explained myself well, what I'm saying is that this text should only appear if your `currentAccountHasPT`, because otherwise the text that appears will be wrong sometimes.

Reply:
I've changed it to saOnOtherAccountJourney

Reply:
No if an account has already been assigned PT they can not then continue and assign a PT enrolment to their current account. The only way they can continue is if a enrolment hasn't already been assigned, i.e the case where they don't recognise the credential that has a SA enrolment.

Reply:
<img width="1526" alt="Screenshot 2022-04-13 at 10 59 00" src="https://user-images.githubusercontent.com/18244373/163153925-9b494ee9-654b-4bfa-925a-d74f029857f6.png">